tox.ini

```diff
@@ -77,6 +78,8 @@ deps =
     figure-!devdeps: astropy==4.0.1.post1
     figure-!devdeps: matplotlib==3.3.2
 
+    build_docs: git+https://github.com/Cadair/sphinx-gallery@skip_on_fail#egg=sphinx-gallery
+
```

Comment:
We don't technically need this here as the tox build runs with `-W` so the behaviour is unchanged with or without the branch.